### PR TITLE
Update modules.mixer

### PR DESCRIPTION
**Pull request by Stefan-Kohl**
Added Gabor Horvath as maintainer of the StructureDescriptions module.

### CODE REVIEW

File: Contacts/People/modules.mixer [hunk 1]
```diff
@@ -499,9 +499,10 @@
   </li>
 
   <li> <strong>Structure Descriptions for Finite Groups</strong>
-       <em> &nbsp;&nbsp;<br/>by Stefan Kohl, Markus P�schel and
+       <em> &nbsp;&nbsp;<br/>by Stefan Kohl, Markus Püschel and
        Sebastian Egner,<br />
        maintained by 
+       <mixer person="Gabor Horvath" data="name_link_city"/> and
```
Comment (alex-konovalov) [inline Contacts/People/modules.mixer]: Please edit this PR leaving only one change which adds the line `<mixer person="Gabor Horvath" data="name_link_city"/>`. First of all, it's not a good practice to mix in one commit actual editing with changing encoding/whitespaces/formatting etc. Second, it's GitHub displaying umlauts in the wrong way, but when Mixer builds html from the source, they all look correctly. I've fetched your PR onto my machine (thanks to @fingolfin's tip about `fetch-pr`) and built the website with your changes and then all umlauts are not displayed properly.

**Comment by fingolfin** [comment]
So @alex-konovalov basically said it all, but let me add this technical bit:

The file says at its top that it is using the LATIN 1 encoding. This PR leaves that marker, but changes the actual encoding to UTF-8, which is incorrect.

We could consider changing the encoding to UTF-8, but then the marker at the file *must* be changed to reflect that. And even then, I am not sure whether the Mixer tool handles this correctly (i.e. some files using latin1, some other utf8). I am not saying it doesn't, just that it should be tested.

File: Contacts/People/modules.mixer (hunk 1 above)
Comment (Stefan-Kohl) [inline Contacts/People/modules.mixer]: The web interface doesn't seem to allow omitting the change of encoding.

**Comment by Stefan-Kohl** [comment]
So the file is automatically recoded from LATIN 1 to UTF-8, and the marker is left the same
-- I'd say this looks like a bug in the software which should be fixed.

**Comment by fingolfin** [comment]
Indeed, this looks like a bug in the GitHub webeditor. We cannot fix that, only report a bug to them (which I will) -- but in the meantime, you can create pull requests as with gap itself, i.e. via the command line. And I guess if we change the encoding of all files from latin1 to UTF-8, that would avoid that bug, too. That would probably a good idea on the long run, too. I'll file an issue about it with us.

**Comment by alex-konovalov** [comment]
This PR may now be closed. Instead of merging it and cleaning up encoding changes, I've changed only one line that adds @hungaborhorvath to maintainers in https://github.com/gap-system/GapWWW/commit/5dd343ba9f3abc40abc5228a4378ef51f3fe6ef9.

**Pull request closed without merging**